src/main/java/com/example/mylib/MyLib/rest/LibraryRestController.java

```diff
@@ -1,14 +1,81 @@
 package com.example.mylib.MyLib.rest;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.example.mylib.MyLib.model.Book;
+import com.example.mylib.MyLib.repositories.BookRepository;
+import com.example.mylib.MyLib.repositories.CustomerRepository;
+import com.example.mylib.MyLib.rest.exceptions.AuthorNotFoundException;
+import com.example.mylib.MyLib.rest.exceptions.CustomerNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Created by hp on 2017-08-10.
  */
+
+
+@RequestMapping("/library")
 @RestController
 public class LibraryRestController {
 
+    private CustomerRepository customerRepository;
+    private BookRepository bookRepository;
+
+    @Autowired
+    public LibraryRestController(CustomerRepository customerRepository, BookRepository bookRepository){
+        this.customerRepository = customerRepository;
+        this.bookRepository = bookRepository;
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "/{customerFirstName}/{customerLastName}")
+    public Book getBooksRentedByCustomer(@PathVariable String customerFirstName, @PathVariable String customerLastName){
+        validateCustomer(customerFirstName, customerLastName);
+
+        return bookRepository.findByBookOwner(customerRepository.findByFirstNameAndLastName(customerFirstName, customerLastName));
```

Comment:
This is quite hard to read

Consider an architecture where, the services do not depend on multiple repositories but on multiple services.